shared/routes/Repl/index.html

```diff
@@ -106,9 +107,23 @@
 		display: none;
 	}
 
+	h2 {
+		margin: 1em 0 0 0;
+		padding: 0 8px;
+		font-weight: 500;
+		font-size: 1.2em;
+	}
+
 	@media (min-width: 768px) {
+		.show-if-mobile {
+			display: none;
+		}
+
 		.repl-container {
+			min-height: auto;
 			height: calc(100vh - 4em);
+			background-color: white;
```

Comment:
Not if you use [currentcolor](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword).